creation/web_base/setup_x509.sh

```diff
@@ -441,7 +441,9 @@ _main() {
         gconfig_add GLIDEIN_CONDOR_TOKEN "$GLIDEIN_CONDOR_TOKEN"
         gconfig_add GLIDEIN_CONDOR_TOKEN_ORIG "$GLIDEIN_CONDOR_TOKEN_ORIG"
         out=$(gconfig_get TRUST_DOMAIN "${glidein_config}")
-        if [[ -z "$out" ]]; then
+        if [[ -n "$out" ]]; then
+            cred_updated+=idtoken,
```

Comment:
Is it intended to have the comma at the end of the line `cred_updated+=idtoken,`  ?

Reply:
yes, it has to, it is a list of values